### ISSUE TITLE
Changes to Net Tests section of website

### ISSUE THREAD

**Issue by agrabeli** [TheTorProject/ooni-web#81]
I propose the following changes to the Net Tests section of OONI's website (https://ooni.torproject.org/nettest/):

1. Replace Net Tests (title of webpage) with **Tests* (To the normal user, "net tests" might potentially be quite confusing).

2. List the test descriptions under the following categories (and include the following short descriptions next to each one):

**Core OONI tests:**

 * **Web connectivity:** Examine whether and how websites are blocked 

* **HTTP header field manipulation:** Examine whether systems that could be responsible for censorship and/or surveillance are in your network

* **HTTP invalid request line:** Examine whether systems that could be responsible for censorship and/or surveillance are in your network

**Circumvention tool tests:**

* Vanilla Tor: Examine whether the Tor network is blocked in your network

* Tor Bridge Reachability: Examine whether Tor bridges - used to circumvent the blocking of the Tor network - are blocked in your network

* Psiphon: Examine whether Psiphon is reachable/blocked in your network

* Lantern: Examine whether Lantern is reachable/blocked in your network

* OpenVPN: Examine whether OpenVPN is reachable/blocked in your network

**Other OONI network measurement tests:**

* HTTP Host

* DNS consistency...

etc etc etc 

In short, all test descriptions would be grouped under 3 categories:

1. Core OONI tests

2. Circumvention tool tests

3. Other OONI network measurement tests

Of course, we could reconsider the name of these groups, but this is a general idea. Then we can include a 1 line description of each test, and upon clicking on the test name the user will be presented with the full description of that test.

Thoughts?

**Comment by agrabeli** [comment]
This was implemented during OONI's hackathon in Berlin.

**Comment by hellais** [comment]
see: https://github.com/TheTorProject/ooni-web/pull/87
